### ISSUE TITLE
Performance with videos

### ISSUE THREAD

**Issue by julioag** [bilal1031/expo-insta-story#2]
Hello @bilal1031,

I have been using your library for a while and i'm having some performance issues with videos. ¿Are you handling some preload logic by any chance? I implemented in my project file downloads so it should access a local source instead but still the performance hasn't change much. Do you have recommendations to improve the loading?

Thank you in advance for the library!

**Comment by bilal1031** [comment]
@julioag, Yes with expo-av the video loading does have performance issues. I would recommend to integrate [react-native-video](https://www.npmjs.com/package/react-native-video) into the library with expo-av to get the performance you are looking for.
